### PR TITLE
Added copy/paste functionality to hogwarts 'columns' and 'group by'

### PR DESCRIPTION
**Pull request by reesercollins**
### ADDITIONAL INFORMATION

- [x] Has associated issue: https://cccs.atlassian.net/browse/CLDN-1432 https://cccs.atlassian.net/browse/CLDN-1433
- [ ] Required feature flags:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by cccs-Dustin — COMMENTED** [review]
This PR seems to meet the requirements set out by the CLDN-1433 ticket very well. However, does it fulfill the requirements in the CLDN-1432 ticket? I am specifically referring to one of the deliverables it asks for:

> Question:
What can/should happen if an invalid column name (e.g., a column name that doesn’t exist in the selected dataset) is copy-pasted into the field as part of a list?
> - Simon Huneault Leblanc ‘s preference is to have the culprit column be highlighted in red and, if the user proceeds to execute the query, the invalid column is ignored and the column runs successfully. We’ll see how close we can get, since the UI doesn’t have this kind of behaviour currently. 
> - A warning message after the valid columns are pasted successfully and the invalid columns are dropped is the second best option.

When I tested the code, it did not seem to throw any kind of error/warning when I tried pasting a column name that does not exist within the dataset.

File: superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/plugin/controlPanel.tsx [hunk 1]
```diff
@@ -228,8 +241,17 @@ const config: ControlPanelConfig = {
         [
           {
             name: 'columns',
-            override: {
-              visibility: isRawMode,
```
[the remark this answers is not in the document]
Comment (reesercollins) [reply superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/plugin/controlPanel.tsx:232]: Yes

[the remark this answers is not in the document]
Comment (cccs-joel) [reply superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/plugin/controlPanel.tsx:232]: Then this is a problem. The Columns widget shouldn't show up in Aggregate mode.

[the remark this answers is not in the document]
Comment (cccs-Dustin) [reply superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/plugin/controlPanel.tsx:232]: From my testing, the "Columns" control does not show up in the Aggregate mode. See screenshots below:
Aggregate
Raw Records

[the remark this answers is not in the document]
Comment (reesercollins) [reply superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/plugin/controlPanel.tsx:232]: Sorry, I misunderstood your comment @cccs-joel. The visibility check is still there, just moved.

File: superset/translations/en/LC_MESSAGES/messages.json [hunk 2]
```diff
@@ -2034,7 +2034,8 @@
       "Check to include time origin dropdown": [""],
       "Limit selector values": [""],
       "These filters apply to the values available in the dropdowns": [""],
-      "Time-series Table": [""]
+      "Time-series Table": [""],
+      "Copy the contents of this control": [""]
```
Comment (cccs-joel) [inline superset/translations/en/LC_MESSAGES/messages.json:2038]: I would put contents singular here, no s.

File: superset-frontend/src/explore/components/ControlHeader.jsx [hunk 3]
```diff
@@ -70,6 +72,17 @@ class ControlHeader extends React.Component {
               />{' '}
             </span>
           )}
+          {this.props.canCopy && (
+            <span>
+              <InfoTooltipWithTrigger
+                label={t('copy')}
+                tooltip={t('Copy the contents of this control')}
```
Comment (cccs-joel) [inline superset-frontend/src/explore/components/ControlHeader.jsx:79]: Contents with an s?

**Review by cccs-joel — APPROVED** [review]

**Pull request merged**